src/Adapters/DropboxConnector.php

```diff
@@ -50,23 +50,23 @@ public function connect(array $config)
      */
     protected function getAuth(array $config)
     {
-        if (!array_key_exists('token', $config) || !array_key_exists('app', $config)) {
-            throw new InvalidArgumentException('The dropbox connector requires authentication.');
+        if (!array_key_exists('token', $config)) {
+            throw new InvalidArgumentException('The dropbox connector requires authorization.');
```

Comment:
Can we keep `authentication` please? Keeps it consistent across this package. :)